pages/tutorials/3.0/graphics/text.md

```diff
@@ -7,37 +7,53 @@
+Firstly, because SFML requires _file names_, not font names, and secondly because SFML doesn't have magical access to your system's font folder.
+If you want to load a font, you will need to include the font file with your application just like every other resource (images, sounds, ...).
 
-The `loadFromFile` function can sometimes fail with no obvious reason. First, check the error message that SFML prints to the standard output (check the console). If the message is unable to open file, make sure that the _working directory_ (which is the directory that any file path will be interpreted relative to) is what you think it is: When you run the application from your desktop environment, the working directory is the executable folder. However, when you launch your program from your IDE (Visual Studio, Code::Blocks, ...) the working directory might sometimes be set to the _project_ directory instead. This can usually be changed quite easily in the project settings.
+The `loadFromFile` function or the corresponding constructor can sometimes fail with no obvious reason.
+First, check the error message that SFML prints to the standard output (check the console).
+If the message is unable to open file, make sure that the _working directory_ (which is the directory that any file path will be interpreted relative to) is what you think it is: When you run the application from your desktop environment, the working directory is the executable folder.
+However, when you launch your program from your IDE (Visual Studio, Code::Blocks, ...) the working directory might sometimes be set to the _project_ directory instead.
+This can usually be changed quite easily in the project settings.
 
 You can also load a font file from memory (`loadFromMemory`), or from a [custom input stream](https://www.sfml-dev.org/tutorials/2.6/system-stream.php "Input streams tutorial") (`loadFromStream`).
+Corresponding constructors exist with the same parameters that throw an exception upon failure.
 
-SFML supports most common font formats. The full list is available in the API documentation.
+SFML supports most common font formats.
+The full list is available in the API documentation.
 
-That's all you need to do. Once your font is loaded, you can start drawing text.
+That's all you need to do.
+Once your font is loaded, you can start drawing text.
 
 ## Drawing text
 
-To draw text, you will be using the [`sf::Text`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Text.php "sf::Text documentation") class. It's very simple to use:
+To draw text, you will be using the [`sf::Text`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Text.php "sf::Text documentation") class.
+It's very simple to use:
 
 ```cpp
-sf::Text text;
+sf::Text text(font); // a font is required to make a text object
 
 // select the font
-text.setFont(font); // font is a sf::Font
+text.setFont(anotherFont); // anotherFont is an sf::Font
```

Comment:
I'd remove this bit, given that it's not relevant for the initial introduction to creating a text object